arch/arm/boot/dts/msm8974-v2.dtsi

```diff
@@ -123,7 +123,7 @@
 		<1880000 2068000>,
 		<3008000 3309000>,
 		<3760000 4136000>,
-		<4468000 2457000>;
+		<4468000 2457600>;
```

Comment:
Neither of these files are used.  OnePlus separates out their dts files into arch/arm/boot/dts/project

In that directory, 14001_HW is for OPO, and 15055_HW is for OPX.